### DISCUSSION TITLE
How to do query of db by date?

### DISCUSSION THREAD

**Discussion by designlook** [appwrite/appwrite#1186] [Q&A]
I created a wildcard field called createdAt and it's Date.now() when I create a record (example: 2021-05-25T06:43:27.241Z)

When I do a query, how would I filter it by something like createdAt < Date.now() ?

**Comment by Duramax2018** [comment]
What all needs to be done. Would u be able to call me Sir

**Reply by designlook** [reply]
> I created a wildcard field called createdAt and it's Date.now() when I create a record (example: 2021-05-25T06:43:27.241Z)
> 
> When I do a query, how would I filter it by something like createdAt < Date.now() ?

I was able to get it working I think. Instead of saving date like new Date, I use Date.now()

var cutoff = Date.now() - 2 * 24 * 60 * 60 * 1000 // 2 days ago
appwrite.database.listDocuments('60axxxxxxxx8', [`createdAt>${cutoff}`])

**Comment by eldadfux** [comment]
@designlook you got it right, store dates as unix time stamps and use common operators to query it.
